core/src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub fn init(v8_flags: Option<Vec<String>>) {
 /// Options for `JSTime::new`.
 pub struct Options {
     pub snapshot: Option<&'static [u8]>,
-    taking_snapshot: bool,
+    pub taking_snapshot: bool,
```

Comment:
This change was necessary to squelch a warning about field assignment after initializing struct using default

I'm not convinced this should have been made public, probably a better way to do this no?

```
warning: field assignment outside of initializer for an instance created with Default::default()
  --> cli/main.rs:36:5
   |
36 | /     options.snapshot = Some(include_bytes!(concat!(
37 | |         env!("OUT_DIR"),
38 | |         "/snapshot_data.blob"
39 | |     )));
   | |________^
   |
   = note: `#[warn(clippy::field_reassign_with_default)]` on by default
note: consider initializing the variable with `jstime_core::Options { snapshot: Some(include_bytes!(concat!(
        env!("OUT_DIR"),
        "/snapshot_data.blob"
    ))), ..Default::default() }` and removing relevant reassignments
  --> cli/main.rs:35:5
   |
35 |     let mut options = jstime::Options::default();
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#field_reassign_with_default

warning: 1 warning emitted
```

Reply:
I would just add `#[allow(clippy::field_reassign_with_default)]` for the moment, the `taking_snapshot` field definitely shouldn't be public

Reply:
I updated code so we can use a `new` constructor to instantiate struct without exposing the internals